tasks/main.yml

```diff
@@ -113,6 +172,30 @@
     state: restarted
   when: apache_modules.changed or koha_plack_result.changed or apache_conf.changed
 
+
+
+- name: make in sshd_config PermitRootLogin to be off
```

Comment:
Same as with apache hardening commit.

Reply:
removed but stored, I will push this hardening code in another role on which we will decide.